Src/library/net/network/socket/network_stream_socket.e

```diff
@@ -79,30 +81,30 @@ feature -- Initialization
 			end
 		end
 
+	make_server_by_address_and_port (a_address: INET_ADDRESS; a_port: INTEGER)
```

Comment:
Making sure that if you see listen to IP 10.0.10.1, an incoming connection on 10.0.10.2 would not be accepted.